rebuild-for-child.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
Goes away once we get this to be self-hosting on Jenkins and are almost ready to merge this branch, right?

Reply:
@602p we do want to delete this, right?  Otherwise is this branch ready to merge?